projects/micro-editor.github.io/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/zyedidia/micro/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: zyedidia/micro/tags
+  strip: /^v/
+
+provides:
+  - bin/micro
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    go.dev: ^1.16
+    git-scm.org: '*'
+  script: |
+    git init
+    git add README.md
+    git config --global user.email "no-reply@tea-xyz.com"
+    git config --global user.name "Tea Build"
+    git commit -m "Fake commit to set the version: {{ version }}"
+    git tag v{{ version }}
+    if test "{{hw.platform}}" = "linux"; then
+        # https://github.com/docker-library/golang/issues/402#issuecomment-982204575
+        sed -i.bak -e 's/^ADDITIONAL_GO_LINKER_FLAGS = /ADDITIONAL_GO_LINKER_FLAGS = "-buildmode=pie" /' Makefile
+        rm Makefile.bak
+    fi
+    make build
+    mkdir -p "{{ prefix }}"/bin
+    mkdir -p "{{ prefix }}"/share/man/man1
+    install -m755 micro "{{ prefix }}"/bin
+    cp assets/packaging/micro.1 {{ prefix }}/share/man/man1
```

Comment:
This doesn't seem impossible. Looking https://github.com/zyedidia/micro/blob/master/tools/compile-linux.sh, it looks like they just bypass their own Makefile when building:

`go build -ldflags "-s -w -X main.Version=$1 -X main.CommitHash=$HASH -X 'main.CompileDate=$DATE'" -o micro-$1/micro ./cmd/micro`

Unless that creates a ton more work, that's probably better.